www/assets/js/lib/search.ts

```diff
@@ -537,7 +537,7 @@ export const courseJSONToLearningResource = (
   run_title: null,
   run_slug: null,
   content_type: null,
-  url: `/courses/${name}/`,
+  url: `/${name}/`,
```

Comment:
There are a few other places in this file where `courses/` is prepended.  I wonder if as a temporary measure we should have some function that adds `courses` if not there, until the open-discussions change has been deployed and the index fully updated, to avoid any downtime?

```javascript
url: `${name.startsWith("courses") ? "/" : "/courses/"}${name}/`
```